tornado/test/web_test.py

```diff
@@ -1,7 +1,8 @@
 from __future__ import absolute_import, division, print_function
+
 from tornado.concurrent import Future
 from tornado import gen
-from tornado.escape import json_decode, utf8, to_unicode, recursive_unicode, native_str, to_basestring
+from tornado.escape import json_decode, utf8, to_unicode, recursive_unicode, native_str, to_basestring  # noqa: E501
```

Comment:
Is there any particular reason you split some of the long imports onto multiple lines and did `noqa` on others?

Reply:
Just a matter of peculiar taste. Some lines were just way too long, and really needed to be wrapped, while others (like this one) were just a bit too long, and wrapping would interrupt the tidy import section.

Maybe you've heard of that saying about vim: "It generally does what you want, unless what you want is consistency" - I suppose it applies to me as well :)

Reply:
... I also came up with the mini-convention for a test file for a particular module wrapping just the imports for that module so they stand out. For example, `web_tests.py` wraps the "from tornado.web import", and it makes some sense for that import to be a different style because it's the stuff under test. (So yeah just random peculiar taste.)

Reply:
Heh, that's fine. I'm kind of partial to the one-long-line style of imports myself (to allow for things like `grep foo | grep -v import` to find all non-import uses of a name), but some of these lines have definitely gotten ridiculous.

It kind of bugs me to place the 14-character `  # noqa: E501` on a line that's "just a bit too long". I guess that's an incentive to try harder to split up long lines and reserve the noqa marker for the cases where it's really not reasonable to do better (or just give up on line-length enforcement).